lib/orderbook/OrderBook.ts

```diff
@@ -191,6 +194,9 @@ class OrderBook extends EventEmitter {
     const pair = this.pairs.get(pairId);
     if (pair) {
       this.pairs.delete(pairId);
+      this.matchingEngines.delete(pairId);
```

Comment:
Again, update the handshake data, re-send `Hello` packet to peers. Otherwise we'll penalize them for keep sending orders of this pair. 

Also, Send order invalidation for all ownOrders in this matchingEngine instance (we might want to introduce invalidating an entire pair instead). Can be done in a separate PR, after merging from master latest order book changes. 

Also, we probably want to trigger order removal events for the client. We shouldn't assume that he's smart enough to trigger his removal logic after calling this. He's relying on these events to fully sync the order book state. Should be done after #123. Can just add a TODO for now.

Reply:
I'll put in a TODO for both of these, thanks.